grpc/src/from_server_proxy.rs

```diff
@@ -109,6 +121,41 @@ pub async fn forward_from_ankaios_to_proto(
 ) {
     while let Some(from_server_msg) = receiver.recv().await {
         match from_server_msg {
+            FromServer::ServerHello(method_obj) => {
+                log::trace!("Received ServerHello from server: {:?}.", method_obj);
+
+                let agent_name = method_obj.agent_name.unwrap_or(String::default());
+
+                if !agent_senders.get_all_agent_names().contains(&agent_name) {
+                    log::warn!(
+                        "Could not send added workloads to started agent. Unknown agent: '{}'",
+                        agent_name
+                    );
+                    return;
```

Comment:
The return is not a good idea here and would break the loop handling messages to agents and clis.
What we want to achieve is to not call the code below. Maybe you can use some `if let Some()` instead

Reply:
Fixed this.